uint8 ADD=0
uint8 DELETE=1
uint8 DELETE_ALL=2
uint8 VALIDATE=3
uint8 INVALIDATE=4
int32 action
algorithm_msgs/robot_state_base robot_base
